api/routes/affiliations/get.test.js

```diff
@@ -59,24 +62,6 @@ tap.test('GET /states/:stateId/affiliations', async endpointTest => {
       invalidTest.ok(next.calledWith(err), 'pass error to middleware');
     });
 
-    handlerTest.test(
```

Comment:
This test is removed because the actual endpoint function no longer controls this access because it has been shelled out to the validForState middleware.